src/azure/CodeGeneratorRba.cs

```diff
@@ -35,6 +46,10 @@ public class CodeGeneratorRba : CodeGeneratorRb
         public override async Task Generate(CodeModel cm)
         {
             var codeModel = cm as CodeModelRba;
+            if (codeModel.ApiVersion == null){
+                throw new NullReferenceException("CodeModel doesn't have api-version set, specs in code model have mismatching api-versions.");
+            }
+            generatedFolderName = Path.Combine(codeModel.ApiVersion, "generated");

```

Comment:
minor: Can we use the private property that is defined as `generated` inside `Path.Combine` :) ?